README.md

```diff
@@ -12,25 +12,41 @@ $ export KBC_DEVELOPERPORTAL_PASSWORD=mypassword
 $ export KBC_DEVELOPERPORTAL_URL=https://apps-api.keboola.com # optional
 $ docker pull quay.io/keboola/developer-portal-cli-v2:latest
 ...
-$ docker run --rm -e KBC_DEVELOPERPORTAL_USERNAME=$KBC_DEVELOPERPORTAL_USERNAME -e KBC_DEVELOPERPORTAL_PASSWORD=$KBC_DEVELOPERPORTAL_PASSWORD quay.io/keboola/developer-portal-cli-v2:latest ecr:get-login keboola keboola.my-application
```

Comment:
tohle tady nemá co dělat, separátní PR pls nebo to přihoď k https://github.com/keboola/developer-portal-cli-v2/pull/13